frontend/src/common/styles.js

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  textStyle: {
+    color: '#55565a',
+    fontSize: 15,
+    lineHeight: 24,
+    letterSpacing: 0.15,
```

Comment:
```suggestion
    letterSpacing: .15,
```

we usually don't write the 0 there

Reply:
My eslint suggested me to do that.
```
A leading decimal point can be confused with a dot.eslint(no-floating-decimal)
```
maybe we can disable this rule if you think it's better.

Reply:
Some styleguides suggest to leave out the trailing 0 before decimal point, but it's your choice. I will let you guys decide this! But please be consequent from now on, if you wanna keep the 0-s, it's fine by me. Please discuss.